www/js/main.js

```diff
@@ -435,8 +476,8 @@ angular.module('emission.main', ['emission.main.recent', 'emission.main.diary',
             'height': '100%', 
             'background-color': '#' + color,
             'color': '#fff',
-            'padding': '15px 15px',
-            'width': '50px'
+            'padding-top': '16px',
```

Comment:
for next week, `getAvatarStyle`, `getButtonStyle` and `getIconButtonStyle` should be moved to css, probably with @media sizing? Also `getAvatarStyle` is obsolete, I think